packages/gateway/src/features/registration/schema.graphql

```diff
@@ -535,6 +536,7 @@ type Attachment { # -> DocumentReference
   contentType: String # -> .content.attachment.contentType
   # base64
   data: String # -> .content.attachment.data base64
+  uri: String
```

Comment:
One cool thing we could do for this type is to split it into two types `URLAttachment` and `DataAttachment` so the type wouldn't have to have so many optional fields. Would probably help using it in the frontend too